vqfx/Makefile

```diff
@@ -1,16 +1,31 @@
 VENDOR=Juniper
 NAME=vQFX
-IMAGE_FORMAT=vmdk
-IMAGE_GLOB=*.vmdk
+IMAGE_FORMAT=qcow2
+IMAGE_GLOB=*.qcow2
 
 # match versions like:
 # vqfx10k-re-15.1X53-D60.vmdk
-VERSION=$(shell echo $(IMAGE) | sed -e 's/^vqfx10k-re-\([0-9][0-9]\.[0-9][A-Z][0-9]\+\(\.[0-9]\+\)\?\(-D[0-9]\+\)\)[^0-9].*$$/\1/')
+# New vqfx are named: vqfx-19.4R1.10-re-qemu.qcow2
+VERSION=$(shell echo $(IMAGE) |  sed -e 's/^vqfx-//'|sed -e 's/-re-qemu.qcow2//' )
+
+PFE_BASE_VERSION=$(shell echo $VERSION | sed -e s'/.*//')
+PFE_IMAGE=$(shell ls vqfx-$(PFE_BASE_VERSION)*-pfe-qemu.qcow*)
 
 -include ../makefile-sanity.include
 -include ../makefile.include
 
-# vqfx10k-pfe-20160609-2.vmdk
 # TODO: we should make sure we only copy one PFE image (the latest?), in case there are many
 docker-pre-build:
-	cp vqfx10k-pfe*.vmdk docker/
+	echo "pfe     $(PFE_IMAGE)"
+	cp vqfx*-pfe*.qcow* docker/
+	echo "image   $(IMAGE)"
+	echo "version $(VERSION)"
+
+# mostly copied from makefile.include, i wish this was easier to change
```

Comment:
Why do you need to copy this. What is it that you want to change / inject? Turn whatever that is into a variable?

Reply:
Hi Kristian. I've changed jobs, so probably won't spend anymore time on this. Hope all is well.